### DISCUSSION TITLE
Need default pdf viewer on object detail page while using generated thumbnails

### DISCUSSION THREAD

**Discussion by sayheyakanksha** [CollectionBuilder/collectionbuilder.github.io#135] [Q&A] [answered]
Hi CollectionBuilder Team!

Thank you for creating such an amazing tool. I’m working on a project in my Digital Libraries class using CollectionBuilder, and it's been a great experience so far. I appreciate all the work you’ve done and shared with the community.

I’m seeking advice on a specific issue. I started with CollectionBuilder-GH, and while everything worked well, I noticed PDF objects show an icon on the browse page, which I didn't like and wanted to change. So, I switched to CollectionBuilder-CSV, used the Rake task to generate thumbnails for PDFs, which worked well - it was awesome btw!

However, I realized that on the object detail page, the PDF viewer gets replaced by the small image. The PDF viewer only appears when I click on the image, whereas in CollectionBuilder-GH, the viewer was displayed by default on the detail page, which I preferred.

My question is: How can I configure CollectionBuilder-CSV to generate PDF thumbnails for the browse page but retain the default PDF viewer on the detail page?

**Comment by evanwill** [comment, marked as answer]
@sayheyakanksha yes that is possible. 
Here is [another Discussion about pdf display](https://github.com/orgs/CollectionBuilder/discussions/86#discussioncomment-5610108) with some thoughts. 
But basically what you need to do is slightly modify the file "_layouts/item/pdf.html" in your project. On line 10 of that file you will see the liquid include:
`{% include item/item-thumb.html %}`

replace that with this include instead:
`{% include item/pdf-embed.html %}`

That's it!

Hope your project goes well!

**Reply by sayheyakanksha** [reply]
Hi @evanwill Thank you for such a quick response. That worked perfectly!
